src/network/network.cpp

```diff
@@ -242,7 +242,10 @@ void NetworkTextMessage(NetworkAction action, TextColour colour, bool self_send,
 			/* Show the Client ID for the server but not for the client. */
 			strid = _network_server ? STR_NETWORK_MESSAGE_CLIENT_JOINED_ID :  STR_NETWORK_MESSAGE_CLIENT_JOINED;
 			break;
-		case NETWORK_ACTION_LEAVE:          strid = STR_NETWORK_MESSAGE_CLIENT_LEFT; break;
+		case NETWORK_ACTION_LEAVE:
+			/* Show client identification for the server but not for the client. */
```

Comment:
the sentence 4 lines higher says exactly the same with different words. Could you please sync those up?